<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>598</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>598</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>576</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/Images/Register.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtEdit_Name">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>250</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(157, 180, 255);
border-radius: 20px;
font-size: 12px;
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtEdit_Email">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>320</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(157, 180, 255);
border-radius: 20px;
font-size: 12px;
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtEdit_Phone">
    <property name="geometry">
     <rect>
      <x>638</x>
      <y>250</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(157, 180, 255);
border-radius: 20px;
font-size: 12px;
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtEdit_Age">
    <property name="geometry">
     <rect>
      <x>638</x>
      <y>180</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(157, 180, 255);
border-radius: 20px;
font-size: 12px;
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radbtn_female">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>330</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_StartRecord">
    <property name="geometry">
     <rect>
      <x>819</x>
      <y>469</y>
      <width>160</width>
      <height>66</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 84, 115);
background-image: url();
border-radius: 20px;
font-size: 16px;
font-weight:bold;
color: rgb(255, 252, 255)

</string>
    </property>
    <property name="text">
     <string>Start Recording</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Clear">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>472</y>
      <width>140</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 84, 115);
background-image: url();
border-radius: 20px;
font-size: 18px;
font-weight:bold;
color: rgb(255, 252, 255)</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cancel">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>472</y>
      <width>140</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 84, 115);
background-image: url();
border-radius: 20px;
font-size: 18px;
font-weight:bold;
color: rgb(255, 252, 255)</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radbtn_male">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="msg_Error">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtEdit_ID">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>180</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(157, 180, 255);
border-radius: 20px;
font-size: 12px;
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
